lib/fs/casefs.go

```diff
@@ -393,20 +396,19 @@ func (f *caseFilesystem) checkCaseExisting(name string) error {
 
 type defaultRealCaser struct {
 	cache caseCache
+	fs    Filesystem
+	mut   sync.Mutex
 }
 
-func newDefaultRealCaser(fs Filesystem) *defaultRealCaser {
+type caseCache = *lru.TwoQueueCache[string, *caseNode]
```

Comment:
This confused me up until reading this line, because the `map[fskey]caseCache` looks like a map of something not-pointer-shaped, and yet here we are with a pointer. I think I'd prefer the alias to not include the pointer star, and have that be visible at the places it's used instead...